### PR TITLE
Jd 2022 08 bootstrap 5 update

### PR DESCRIPTION
**Pull request by goapunk**
Todo:

- [ ] fix grey footer color

### CODE REVIEW

**Review by khamui — APPROVED** [review]
nice! lgtm.

about the todo. Do you want to add it to this PR or in another?

File: apps/core/static/scss/_variables.scss [hunk 1]
```diff
@@ -58,3 +58,5 @@ $container-max-widths: (
 );
 
 $transition-duration: 0.3s;
+
+$border-color: rgba(6, 6, 6, 0.15);
```
Comment (phillimorland) [inline apps/core/static/scss/_variables.scss:62]: we should use hex colours in variables

File: apps/core/templates/footer.html [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
 <footer id="footer" class="footer">
     <nav class="navbar navbar-expand-md">
-        <div class="container-fluid">
+        <div class="container">
```
Comment (phillimorland) [inline apps/core/templates/footer.html:9]: is there a reason we stop using container fluid here?

**Review by phillimorland — COMMENTED** [review]
would just check that last commit @khamui everything else looks good

File: apps/core/templates/footer.html (hunk 2 above)
Comment (khamui) [reply apps/core/templates/footer.html:9]: without the fluid it has a little space on the start and end sides. Which matches better with the design. But maybe should be done with padding right @phillimorland ?

**Review by phillimorland — APPROVED** [review]
cool looking good!

**Pull request merged**